Documentation/doc-dataframe.lisp

```diff
@@ -0,0 +1,110 @@
+(defpackage #:eu.turtleware.polyclot.example
+  (:use #:clim-lisp #:alexandria #:eu.turtleware.polyclot))
+(in-package #:eu.turtleware.polyclot.example)
+
+;;; This example file follows the data frame tutorial
+;;; snippets. SHOW-DATA-FRAME is a helper utility for data frames to
+;;; visualise them. If you load this file it will open a few windows,
+;;; it is advised to evaluate top level forms step by step when
+;;; reading the tutorial.
+
+(defun show-data-frame (data-frame
+                        &aux (clim-stream (clim:open-window-stream)))
+  (setf (clime:stream-text-margins clim-stream) '(:left 10 :top 5))
```

Comment:
I'm using the version of McCLIM in quicklisp. In this version, `'(:left 10 :top 5)` is not of type margin-spec. I have to edit it to `'(:left (:absolute 10) :top (:absolute 5))`.

Reply:
next version will allow atomic values. either way it is not a file which is loaded with the system.